prompting/rewards/pipeline.py

```diff
@@ -1,27 +1,7 @@
 from typing import List
 
 from prompting.tasks import TASKS
-from prompting.rewards import (
-    BaseRewardModel,
-    RougeRewardModel,
-    DiffRewardModel,
-    RelevanceRewardModel,
-    FloatDiffModel,
-    DateRewardModel,
-    OrdinalRewardModel,
-    StreamingRewardModel
-)
-
-REWARD_MODELS = {
-    "rouge": RougeRewardModel,
-    "relevance": RelevanceRewardModel,
-    "diff": DiffRewardModel,
-    "float_diff": FloatDiffModel,
-    "date": DateRewardModel,
-    "ordinal": OrdinalRewardModel,
-    "streaming": StreamingRewardModel,
-}
-
+from ..rewards import BaseRewardModel, REWARD_MODELS
```

Comment:
Generally relative imports are discouraged in python - it's not as strongly discouraged as it once was but good to keep in mind. This happens a lot throughout the codebase so I won't comment on it every time, but it's good to keep in mind for future development!

Reply:
Fair enough - my condolences. I've been there